common/protob/messages-neo.proto

```
@@ -0,0 +1,63 @@
+    optional string address = 1;                // Neo address as hex-encoded string
+    optional bytes public_key = 2;               // public key used to generate the address
+}
+
+/**
+ * Request: ask device to sign Neo transaction
+ * @start
+ * @next NeoSignedTx
+ */
+message NeoSignTx {
+    repeated uint32 address_n = 1;         // BIP-32 path to derive the key from master node
+    required bytes raw_tx = 2;              // serialized raw transaction
```

Buf (1.47.2)

Field named %!q(MISSING) should not be required.

(FIELD_NOT_REQUIRED)